Public/Src/Cache/ContentStore/Library/Service/LocalContentServerBase.cs

```diff
@@ -7,23 +7,24 @@
 using System.Diagnostics;
 using System.Diagnostics.ContractsLight;
 using System.Linq;
+using System.Net;

```

Comment:
Do we need all these extra libraries now?

Reply:
We only needed to add `System.Net`. I just sorted the others.

---
In reply to: [274158652](https://github.com/Microsoft/BuildXL/pull/120#discussion_r274158652) [](ancestors = 274158652)

Reply:
Gotcha. Cool!